vllm/model_executor/layers/quantization/kernels/scaled_mm/grouped_cutlass.py

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Optional, Tuple
+
+import torch
+
+from vllm import _custom_ops as ops
+from vllm.model_executor.layers.quantization.utils import replace_parameter
+from vllm.platforms import current_platform
+
+from .GroupedMMLinearKernel import (GroupedMMLinearKernel,
+                                    GroupedMMLinearLayerConfig)
+
+
+class CutlassGroupMMLinearKernel(GroupedMMLinearKernel):
+
+    @classmethod
+    def get_min_capability(cls) -> int:
+        return 75
+
+    @classmethod
+    def can_implement(
+            cls, c: GroupedMMLinearLayerConfig) -> Tuple[bool, Optional[str]]:
+
+        if (not current_platform.is_cuda() and not current_platform.is_cpu()):
+            return False, "CutlassScaledMM requires running on CUDA or CPU."
```

Comment:
Not supported on CPUs:
```suggestion
        if not current_platform.is_cuda():
            return False, "CutlassScaledMM requires running on CUDA ."
```